src/site/content/en/secure/what-is-mixed-content/index.md

```diff
@@ -0,0 +1,157 @@
+
+The following Glitch demo contains examples of passive mixed content.
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/passive-mixed-content?path=index.html&previewSize=100"
+    title="passive-mixed-content on Glitch"
+    allow="encrypted-media"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+If passive mixed content is present most browsers will indicate in the URL bar that the page is not secure,
+even when the page itself was loaded over HTTPS.
+
+Until recently passive mixed content was loaded in all browsers,
+as to block it would have broken many websites.
+This is now beginning to change and so it is vital to update any instances of mixed content on your site.
+
+Chrome is currently rolling out automatic upgrading of passive mixed content where possible.
+Automatic upgrading means that if the asset is available over HTTPS, but has been hardcoded as HTTP,
+the browser will load the HTTPS version. If no secure version can be found the asset will not load.
+
+When mixed content is detected, or auto-upgrading has occurred, this will be noted in the Chrome DevTools console.
+
+<figure class="w-figure">
+  <img class="w-screenshot"
+      src="passive-mixed-content.jpg"
+      alt="Chrome DevTools showing the warnings displayed when mixed content is detected and upgraded">
+</figure>
+
+### Active mixed content
+
+Active mixed content poses a greater threat than passive mixed content.
+An attacker can intercept and rewrite active content,
+thereby taking full control of your page or even your entire website.
+This allows the attacker to change anything about the page,
+including displaying entirely different content,
+stealing user passwords or other login credentials,
+stealing user session cookies,
+or redirecting the user to a different site entirely.
+
+Due to the severity of this threat,
+many browsers already block this type of content by default to protect users,
+but functionality varies between browser vendors and versions.
+
+The following contains examples of active mixed content:
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/active-mixed-content?previewSize=100"
+    title="active-mixed-content on Glitch"
+    allow="encrypted-media"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+[Load the example over HTTP](http://active-mixed-content.glitch.me/) and you will see the content that has been blocked.
+
+Blocked content will be detailed in the Chrome DevTools console.
+
+<figure class="w-figure">
+  <img class="w-screenshot"
+      src="active-mixed-content.jpg"
+      alt="Chrome DevTools showing the warnings displayed when active mixed content is blocked">
+</figure>
+
+## The mixed content specification
+
+Browsers follow the [mixed content specification](https://w3c.github.io/webappsec-mixed-content/),
```

Comment:
Optional: We are in the process of writing a new Level 2 version of mixed content (https://w3c.github.io/webappsec-mixed-content/level2.html), which will put autoupgrades in the spec, and which eventually will completely replace MIX. It might be worth mentioning its existence int his section (though since it's a WIP we can also edit this section once MIX2 is done, either way works for me).